lib/shared/shared-asset-bundler.ts

```diff
@@ -47,17 +51,75 @@ export class SharedAssetBundler extends Construct {
   constructor(scope: Construct, id: string, sharedAssets: string[]) {
     super(scope, id);
     this.sharedAssets = sharedAssets;
+    // Check if we can do local bundling
+    if (!this.localBundlerTest()) {
+      // if not, then build Apline from local definition
+      this.container_image = DockerImage.fromBuild(path.posix.join(__dirname, "alpine-zip"));
```

Comment:
shouldn't the `containerImage` be a singleton to avoid rebuilding the container for every use of a SharedAssetBundler construct?

Reply:
makes sense to me, but the [current implementation](https://github.com/aws-samples/aws-genai-llm-chatbot/pull/320/commits/d7b1c6a4b723a5c71112219fc804c5c39fc87cf0#diff-36fe271369c32e3bbc80d20fe50ebe4be930a8015edcb6c946fb8edc6e62e4c1L60) builds a new image with each invocation of method `bundleWithAsset(...)`, so I thought perhaps a Singleton was problematic for some reason? I'm not sure why it would be a problem, but I stopped short of Singleton because this change improved build time and my knowledge of code base is limited.

Reply:
The current implementation should be improved :-)

Reply:
done